clang/include/clang/3C/RewriteUtils.h

```diff
@@ -118,11 +118,21 @@ class FunctionDeclReplacement
 
     // If rewriting Parameters, stop at the right parenthesis of the parameters.
     // Otherwise, stop after the return type.
-    // Note: getFunctionDeclarationEnd is used instead of getRParenLoc so that
-    // itypes are deleted correctly when --remove-itypes is used.
```

Comment:
Is `--remove-itypes` still available? I assume we don't want it anymore, and the new code here doesn't use it, so if it lingers we could remove it here or in a later pull request.

Reply:
Remove itypes still works on the `5c` build, although we'll probably want to update it to include removing unnecessary casts. This comment is removed because the conditional below it handles overwriting the bounds and interop type expression correctly in the first two branches without needing to fall through to `getFunctionDeclarationEnd`